cmd/kots/cli/install.go

```diff
@@ -1076,3 +1099,19 @@ func checkPreflightResults(response *handlers.GetPreflightResultResponse, skipPr
 
 	return true, nil
 }
+
+func slugInLicenseChannels(slug string, license *kotsv1beta1.License) bool {
+	for _, channel := range license.Spec.Channels {
+		if channel.ChannelSlug == slug {
+			return true
+		}
+	}
+	return false
+}
+
+func haveMultiChannelLicense(license *kotsv1beta1.License) bool {
+	if license == nil {
+		return false
+	}
+	return len(license.Spec.Channels) > 0
```

Comment:
mind adding a comment here that the existence of the `channels` field in the license is what determines if the license is multi channel or not? took me a second to realize why this is `> 0` and not `> 1`.